cmd/kubecertinit/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+
+	"github.com/ekspand/trusty/kubeca/certinit"
+)
+
+func main() {
+	var kubeConfig string
+	r := &certinit.Request{}
+	flag.StringVar(&kubeConfig, "kubeconfig", "", "(optional) path to kubeconfig file")
+	flag.StringVar(&r.Namespace, "namespace", "", "namespace as defined by pod.metadata.namespace")
```

Comment:
Just a note,, in kubeca/main.go, the fourth parameters start with capital case. Maybe for consistency lets make messages here also start with capital case.